src/app/publications/international-conferences-with-proceedings.mdx

```diff
@@ -4,6 +4,9 @@
 1.  S Liu, N Benchasattabuse, D QC Morgan, M Hajdušek, S J. Devitt, R Van Meter,
     **A Substrate Scheduler for Compiling Arbitrary Fault-tolerant Graph States**,
     2023 IEEE International Conference on Quantum Computing and Engineering (QCE),   Sep 2023
+1.  Sane Bernard Ousmane ,  Van Meter Rodney ,  Hajdusek Michal,
```

Comment:
`Hajdušek` the "s"  should be special one?

Reply:
I made the correction